src/screens/how-to-help/raise-awareness.css

```diff
@@ -0,0 +1,27 @@
+.link-copied-toast[data-state='open'] {
+  animation: slideUp 150ms ease-out forwards;
+}
+
+.link-copied-toast[data-state='closed'] {
+  animation: slideDown 150ms ease-in forwards;
+}
+
+@keyframes slideUp {
```

Comment:
You can define keyframes and animations in the Tailwind config. We also use the TailwindCSS Radix plugin (see `lib/`) that allows you to directly target the open and closed states in your classnames:
```tsx
<div className="radix-state-open:animate-slide-up radix-state-closed:animate-slide-down" />
```